packages/@lwc/synthetic-shadow/src/shared/static-html-collection.ts

```diff
@@ -63,60 +56,6 @@ StaticHTMLCollection.prototype = create(HTMLCollection.prototype, {
         },
     },
 
-    // Iterator protocol
```

Comment:
`HTMLCollection` is not an iterable, while `NodeList` is.

Reply:
It would not be a surprise if somebody discovered this and is using it. We would be breaking them. Can we be conservative and hide this deprecation behind a feature flag?